setup.py

```diff
@@ -25,4 +23,7 @@
         ],
     packages = ['isbn'],
     package_dir = {'isbn': 'isbn'},
-    )
+    autosemver={
+        'bugtracker_url': True,
```

Comment:
Normally I reuse the url, putting it on a global and reusing it for the url param and this one (URL + '/issues').